contrib/DEBIAN/zcash.install

```diff
@@ -0,0 +1,3 @@
+usr/bin/zcashd
```

Comment:
@str4d nope, "Alternatively, this install can have the name of the file only without the installation directory when the relative directory path does not change."